File: py/Soal-2/tikus.py
from mamalia import Mamalia

class Tikus(Mamalia) :
    def __init__(self) :
        super().__init__()
        super().setNama("Tikus")
        super().setUkuran("Kecil")
        super().setJumlahKaki(4)
        super().setHabitat("Perumahan")

File: cpp/Soal-2/tikus.cpp
#include "mamalia.cpp"
#pragma once

class Tikus: public Mamalia {
public:
    Tikus() {
        Mamalia::setNama("Tikus");
        Mamalia::setUkuran("Kecil");
        Mamalia::setJumlahKaki(4);
        Mamalia::setHabitat("Perumahan");
    }
};
